example/backends/spidsaml2.py

```diff
@@ -218,38 +219,55 @@ def _metadata_endpoint(self, context):
         conf = self.sp.config
 
         metadata = entity_descriptor(conf)
+ 
         # creare gli attribute_consuming_service
-        cnt = 0
-        for (
-            attribute_consuming_service
-        ) in metadata.spsso_descriptor.attribute_consuming_service:
-            attribute_consuming_service.index = str(cnt)
-            cnt += 1
-
-        cnt = 0
-        for (
-            assertion_consumer_service
-        ) in metadata.spsso_descriptor.assertion_consumer_service:
-            assertion_consumer_service.is_default = "true" if not cnt else ""
-            assertion_consumer_service.index = str(cnt)
-            cnt += 1
-
-        # nameformat patch... tutto questo non rispecchia gli standard OASIS
-        for reqattr in metadata.spsso_descriptor.attribute_consuming_service[
-            0
-        ].requested_attribute:
+        metadata.spsso_descriptor.attribute_consuming_service[0].index = '0'
+        metadata.spsso_descriptor.attribute_consuming_service[0].service_name[0].lang = "it"
+        metadata.spsso_descriptor.attribute_consuming_service[0].service_name[0].text = metadata.entity_id
+        for reqattr in metadata.spsso_descriptor.attribute_consuming_service[0].requested_attribute:
             reqattr.name_format = None
             reqattr.friendly_name = None
 
-        # attribute consuming service service name patch
-        service_name = metadata.spsso_descriptor.attribute_consuming_service[
-            0
-        ].service_name[0]
-        service_name.lang = "it"
-        service_name.text = metadata.entity_id
-
-        # remove extension disco and uuinfo (spid-testenv2)
-        # metadata.spsso_descriptor.extensions = []
+        metadata.spsso_descriptor.assertion_consumer_service[0].index = '0'
+        metadata.spsso_descriptor.assertion_consumer_service[0].is_default = 'true'
+
+        if self.config["sp_config"]["enable_cie"] is True:
```

Comment:
Did you mean "enable ficep"?

Reply:
yes, I manually updated the yaml file from my server and I was wrong....